Extensions/replyviewer.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE ReplyViewer **//
-//* VERSION 0.2 REV G **//
+//* VERSION 0.3 **//
```

Comment:
Would it not actually be 0.2.7? Since it's a bugfix, or is it a feature addition?